test/suite/test_hs06.py

```diff
@@ -47,7 +47,8 @@ class test_hs06(wttest.WiredTigerTestCase):
     conn_config = 'cache_size=50MB,statistics=(fast)'
     session_config = 'isolation=snapshot'
     key_format_values = [
-        ('column', dict(key_format='r')),
+        # The commented columnar tests needs to be enabled once columnar page instantiated is fixed.
+        # ('column', dict(key_format='r')),
```

Comment:
Can this part to disable column store testing be marked some way so it doesn't get lost.  Either put a pointer to the test in a ticket, or have a standard way to mark "tests that need to be reenabled when we get column store working".

Reply:
Created a new ticket to handle it WT-6061. And also mentioned in the test file also.